### PR TITLE
Fixed PHPCS errors.

### PR DESCRIPTION
**Pull request by jibran**
We can test it after #88.

### CODE REVIEW

File: src/Plugin/Condition/DataComparison.php [hunk 1]
```diff
@@ -49,31 +49,37 @@ public function summary() {
...
     $data = $this->getContextValue('data');
     $operator = $this->getContext('operator')->getContextData() ? $this->getContextValue('operator') : '==';
     $value = $this->getContextValue('value');
-    
+
     switch ($operator) {
       case '<':
-        return $data < $value;
+        $result = $data < $value;
+        break;
```
Comment (klausi) [inline src/Plugin/Condition/DataComparison.php]: Why do we need this change? This is fine as is?

Comment (jibran) [inline src/Plugin/Condition/DataComparison.php]: Please see https://travis-ci.org/fago/rules/jobs/29828127#L179

Comment (klausi) [inline src/Plugin/Condition/DataComparison.php]: Ah, so there is a bug in Coder? There is a newline after the case breaking return statement, so this is fine?

Comment (jibran) [inline src/Plugin/Condition/DataComparison.php]: Yeah this is bug. But I couldn't fix it till I add break and I can't add break after return so this is the only way to fix it.

Comment (klausi) [inline src/Plugin/Condition/DataComparison.php]: So let's not "fix" stuff that is not broken. I fear that PHPCS for Drupal is not quite as mature as we would like it to be right now. Let's file bug reports against Coder instead.

**Comment by klausi** [comment]
Squashed and merged, thanks!

**Pull request closed without merging**